tools/wptrunner/wptrunner/browsers/servo.py

```diff
@@ -86,3 +87,30 @@ def executor_browser(self):
             "user_stylesheets": self.user_stylesheets,
             "ca_certificate_path": self.ca_certificate_path,
         }
+
+
+class ServoWdspecBrowser(WebDriverBrowser):
+    # TODO: could share an implemenation with servodriver.py, perhaps
+    def __init__(self, logger, binary="servo", webdriver_args=None,
+                 binary_args=None, host="127.0.0.1", env=None, port=None):
+
+        env = env if env is not None else os.environ.copy()
```

Comment:
Matching what appears earlier in this diff and is easier to read:

```suggestion
        env = os.environ.copy() if env is None else env
```